### ISSUE TITLE
Calling #subscribe alters the observable behavior ?

### ISSUE THREAD

**Issue by ArThoX** [Urigo/meteor-rxjs#19]
Hi

Example 1

```
this.autorun(h => {
    this.subscribe('myCollection', this.filterQuery.get(), {limit: this.recordLimit.get()});
    this.items= MyCollection.find(this.filterQuery.get(), {limit: this.recordLimit.get()});
```

Example 2 (1 line added at the end)

```
this.autorun(h => {
    this.subscribe('myCollection', this.filterQuery.get(), {limit: this.recordLimit.get()});
    this.items= MyCollection.find(this.filterQuery.get(), {limit: this.recordLimit.get()});

    this.items.subscribe();
```

In both examples, ReactiveVars are running in an `autorun` to update the subscription and collection cursor. `this.items` is in both cases rendered with a `ngFor .. | async`.

However, when I increase the limit value `this.recordLimit.set(this.recordLimit.get() + 10)`, I notice that:
- In the first example, new items are painted by my `ngFor`.
- In the second exampe, firs all the items are removed, then all are repainted again.

Is that normal `subscribe()` has this kind of impact ?

**Comment by dotansimha** [comment]
Please try to run:
```js
    this.subscribe('myCollection', this.filterQuery.get(), {limit: this.recordLimit.get()});
    this.items= MyCollection.find(this.filterQuery.get(), {limit: this.recordLimit.get()}).publish().refCount();

    this.items.subscribe();
```

I should turn your Observable into a multi cast Observable, and you should be able to both subscribe and iterate using ngFor.

**Comment by dotansimha** [comment]
Maybe related to https://github.com/Urigo/meteor-rxjs/issues/23

**Comment by dotansimha** [comment]
Closed in #24 . please re-open if you still having issues @ArThoX .
